conftest.py

```diff
@@ -2,6 +2,8 @@
 
 import pytest
 
+import borg.cache
```

Comment:
If we need to monkey patch this we really should consider boxing it into a mutable object.

For now i added warning comments to borg/__init__.py and conftest.py